src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseService.java

```diff
@@ -139,7 +170,7 @@ public ProgrammingExerciseService(ProgrammingExerciseRepository programmingExerc
      * <li>VCS webhooks</li>
      * <li>Bamboo build plans</li>
      * </ul>
-     *
+     * <p>
```

Comment:
```suggestion
     *
```
This `<p>` tag is not closed